binsync/common/controller.py

```diff
@@ -173,10 +174,10 @@ def start_pull_routine(self):
         self.pull_thread.start()
 
     def _check_and_notify_ctx(self):
-        active_ctx = self.active_context()
+        active_ctx_name, active_ctx = self.active_context()
```

Comment:
see my other comment about this, as I don't see why we need both name and ctx. We should only need ctx.

Reply:
agreed